packages/terra-framework-docs/src/terra-dev-site/test/worklist-data-grid/PinnedColumnsWithRowSelection.test.jsx

```diff
@@ -99,16 +99,16 @@ const RowSelection = () => {
       defaultColumnWidth={180}
       ariaLabel="Worklist Data Grid with Pinned Columns and Row Selection"
       hasSelectableRows={hasSelectableRows}
-      onRowSelect={(rowId) => {
-        const newRows = [];
-        const selectedRow = rows.find(row => row.id === rowId);
-        selectedRow.isSelected = !selectedRow.isSelected;
-        rows.forEach(element => {
-          if (element.isSelected) {
-            newRows.push(element.id);
+      onRowSelect={(rowIdsToSelect, rowIdsToUnselect) => {
```

Comment:
Question: Could we possibly not inline this? This seems to have enough going on that it should be a separate function space.